src/HealthKit/Enums.cs

```diff
@@ -564,4 +564,52 @@ public enum HKVO2MaxTestType : long {
 		PredictionSubMaxExercise,
 		PredictionNonExercise,
 	}
+ 
+	[NoWatch, iOS (12, 0)]
+	public enum HKFHIRResourceType {
```

Comment:
You've bound `FHIR` as `FhiR` elsewhere (btw: why isn't that `Fihr`? it seems strange to capitalize the last letter only).